projects/canopy/src/lib/forms/date/date-field.component.ts

```diff
@@ -144,30 +144,30 @@ export class LgDateFieldComponent implements OnInit, ControlValueAccessor, OnDes
       this.ngControl.control.updateValueAndValidity();
     }
 
-    this.valueChanges = this.dateFormGroup.valueChanges.subscribe((date: DateField) => {
-      const day = date.date ? ('0' + date.date).slice(-2) : '';
-      const month = date.month ? ('0' + date.month).slice(-2) : '';
-      const year = date.year ? date.year : '';
-      this.onChange(`${year}-${month}-${day}`);
-    });
+    this.subscriptions.push(
```

Comment:
Andy suggested to write tests for this but I honestly don't know how to do it? I don't really want to assign variables to the subscriptions only for testing purposes. We don't test unsubscriptions anywhere else and I'm not 100% sure I see the benefit.

Reply:
I'd say maybe something to keep an eye on, I think there are ways to do it, but wouldn't hold this PR up for it 👍